gnomad/utils/annotations.py

```diff
@@ -1402,19 +1394,77 @@ def create_subpops(
         "population": "gnomad3:global",
         "ancillaryResults": {
             "popMaxFAF95": {
-                "frequency": ht.popmax.AF.collect()[0],
+                "frequency": ht.popmax.faf95.collect()[0],
                 "confidenceInterval": 0.95,
                 "popFreqID": f"{variant}.{ht.popmax.pop.collect()[0].upper()}",
             },
-            "homozygotes": ht.popmax.homozygote_count.collect()[0],
+            "homozygotes": overall_freq["homozygote_count"].collect()[0],
             "meanDepth": coverage_stat,
         },
-        "subpopulationFrequency": list_of_ancestry_dicts,
     }
 
+    if groups:
+        ret_final["subpopulationFrequency"] = list_of_group_info_dicts
+
     try:
-        ret_json = json.dumps(ret_final)
+        validated_json = json.dumps(ret_final)
     except:
         raise SyntaxError("The dictionary did not convert to a valid JSON")
 
     return ret_final
+
+
+def gnomad_gks(
+    version: str,
+    variant: str,
+    groups: list = None,
+    by_sex: bool = False,
+    vrs_only: bool = False,
+) -> dict:
+    """
+    Call get_gks() for a specified gnomAD release version, variant, and ancestry groups. Splitting by sex or only returning VRS information is optional.
+
+    :param version: String of version of gnomAD release to call .
+    :param variant: String of variant to filter to. Example: 'chr1-1376025-T-C' .
+    :param groups: List of ancestry group appreviations to call. Example: ['amr','nfe','fin'] .
+    :param by_sex: Boolean to pass if you want to return frequecy information for each ancestry group split by chromosomal sex.
+    :param vrs_only: Boolean to pass if you only want VRS information returned - no frequency or other information.
+    :return: result of get_gks as a 'Dictionary containing VRS information (and population and sex if desired) for specified variant'.
+
+    """
+    ht = hl.read_table(
+        f"gs://gcp-public-data--gnomad/release/{version}/ht/genomes/gnomad.genomes.{version}.sites.ht"
+    )
+
+    # Read coverage statistics
+    coverage_ht = hl.read_table(coverage("genomes").path)
+
+    if groups and vrs_only:
+        logger.warning(
+            "You cannot pass both a pops list and vrs_only. Ignoring pops list."
+        )
+        outer_result = get_gks(
```

Comment:
you should only need one line for calling get_gks (not three) and pass in all the parameters:

gks_info = get_gks(
            ht = ht,
            variant = variant,
            coverage_ht = coverage_ht,
            label_name = "gnomAD",
            label_version = version,
            ancestry_groups=ancestry_groups,
            ancestry_group_names=POP_NAMES,
            by_sex=by_sex,
            vrs_only=vrs_only,
        )